### PR TITLE
June 15th 2022 release notes

### PR DESCRIPTION
**Pull request by dannify**
June 15th 2022 release notes

### CODE REVIEW

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+export default BlogPostLayout;
+
+---
+description: todo
+date: 2022-06-15
+---
+
+# June 15, 2022 Release
+
+After a long wait, our date and time components and hooks have been released!! Our DatePicker and Calendar packages support dates and times with configurable granularity in 13 different calendar systems used around the world! They are accessible, time zone aware, and touch friendly. They are also internationalized, supporting locale specific formatting, number systems, calendar systems, hour cycles, and right-to-left languages. Add more... You can find docs for the calendars, date and time fields, and range pickers below.
```
Comment (ktabors) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:20]: Could the "And more..." be "You can read about these features and more in the docs for..."

Comment (dannify) [reply packages/dev/docs/pages/releases/2022-06-15.mdx]: Not done yet. Still in draft.

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 2]
```diff
@@ -0,0 +1,239 @@
...
+## Docs
+* Update W3C APG links - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/3204)
+* Update TableView docs for checkbox/highlight + onAction behavior update - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3205)
+
+## Under construction
+
+Fixes to pre-released components are listed below. Please feel free to try them out, and report any issues you encounter.
+
+* Fix `ListView` shifting height with truncation - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3158)
+* Allow focusing disbaled rows when dragging in `ListView` - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/3181)
```
Comment (LFDanLu) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:60]: sorry bout the PR title, this fix actually prevents focus from reaching the disabled rows if they aren't valid drop targets. Perhaps this should be "ListView drag and drop focus fixes" or something along those lines?

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+date: 2022-06-15
+---
+
+# June 15, 2022 Release
+
+After a long wait, our date and time components and hooks have been released! 🎉.  Our DatePicker and Calendar packages support dates and times with configurable granularity in 13 different calendar systems used around the world! They are accessible, time zone aware, and touch friendly. They are also internationalized, supporting locale specific formatting, number systems, calendar systems, hour cycles, and right-to-left languages. You can find docs for the calendars, date and time fields, and range pickers below.
+
+In addition to these components is a new library [`@internationalized`](https://react-spectrum.adobe.com/internationalized/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/date/index.html).
```
Comment (devongovett) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:22]:
```suggestion
In addition to these components is a new library [@internationalized/date](https://react-spectrum.adobe.com/internationalized/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/date/index.html).
```

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: After a long wait, our date and time components and hooks have been released!! Our DatePicker and Calendar packages support dates and times with configurable granularity in 13 different calendar systems used around the world! They are accessible, time zone aware, and touch friendly. They are also internationalized, supporting locale specific formatting, number systems, calendar systems, hour cycles, and right-to-left languages.
```
Comment (ktabors) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:14]: do we want the double `!!` at the end of the first sentence?

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+date: 2022-06-15
+---
+
+# June 15, 2022 Release
+
+After a long wait, our date and time components and hooks have been released! 🎉.  Our DatePicker and Calendar packages support dates and times with configurable granularity in 13 different calendar systems used around the world! They are accessible, time zone aware, and touch friendly. They are also internationalized, supporting locale specific formatting, number systems, calendar systems, hour cycles, and right-to-left languages. You can find docs for the calendars, date and time fields, and range pickers below.
+
+In addition to these components is a new library [@internationalized/date](https://react-spectrum.adobe.com/internationalized/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/date/index.html).
+
+A special thanks to [@devongovett](https://github.com/devongovett) for his work on all things date related and to our other contributors for this release 🤩
```
Comment (ktabors) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:24]: period/exclamation at the end?

Comment (devongovett) [reply packages/dev/docs/pages/releases/2022-06-15.mdx:24]: so much excitement!!

File: packages/dev/docs/pages/releases/2022-06-15.mdx (hunk 4 above)
Comment (devongovett) [reply packages/dev/docs/pages/releases/2022-06-15.mdx:14]: so much excitement!!

File: packages/dev/docs/pages/releases/2022-06-15.mdx [hunk 6]
```diff
@@ -0,0 +1,240 @@
...
+date: 2022-06-15
+---
+
+# June 15, 2022 Release
+
+After a long wait, our date and time components and hooks have been released!! 🎉.  Our DatePicker and Calendar packages support dates and times with configurable granularity in 13 different calendar systems used around the world! They are accessible, time zone aware, and touch friendly. They are also internationalized, supporting locale specific formatting, number systems, calendar systems, hour cycles, and right-to-left languages. You can find docs for the calendars, date and time fields, and range pickers below.
+
+In addition to these components is a new library [@internationalized/date](https://react-spectrum.adobe.com/internationalized/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/date/index.html).
```
Comment (dannify) [inline packages/dev/docs/pages/releases/2022-06-15.mdx:22]:
```suggestion
In addition to these components is a new library [@internationalized/date](https://react-spectrum.adobe.com/internationalized/date/index.html), that provides locale-aware objects and functions for manipulating dates and times. This library supports time zone and calendar system aware arithmetic, locale-specific queries and conversions, and more in just 8 kB! Learn more about it in our [docs](https://react-spectrum.adobe.com/internationalized/index.html).
```

**Review by devongovett — APPROVED** [review]

**Pull request merged**